test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -1615,6 +1615,29 @@ public void InstallCommand_AuthenticatedV3WithCredentialSavedInConfig()
             }
         }
 
+        /// <summary>
+        /// Unit test that proves non-zero exit code functionality when the number of
+        /// arguments for install command are not appropiate
+        /// </summary>
+        [Fact]
+        public void InstallCommand_WrongArguments()
+        {
+            // prepare
+            string[] args = {
+                "install",
+                "mypackage",
+                "-version",
+                "-outputdirectory",
+            };
+
+            // act & assert
+            using(var test_dir = TestDirectory.CreateInTemp())
+            {
+                var result = Program.MainCore(test_dir, args);
```

Comment:
Instead of using `Program.MainCore` I think is a better idea to use the `CommandRunner`. [Here](https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetVerifyCommandTest.cs#L17) is an example of a test using it.

Reply:
Changed to `CommandRunner`.